pkg/ui/react-app/src/utils/index.ts

```diff
@@ -223,6 +224,12 @@ export const encodePanelOptionsToQueryString = (panels: PanelMeta[]) => {
 export const createExpressionLink = (expr: string) => {
   return `../graph?g0.expr=${encodeURIComponent(expr)}&g0.tab=1&g0.stacked=0&g0.range_input=1h`;
```

Comment:
We could improve this even more in the future by setting `range_input` to `$now - $when_the_alert_has_started_firing` but this is OK for now :muscle: